.github/workflows/test.yml

```diff
@@ -64,3 +65,13 @@ jobs:
           build-source: ${{ steps.emit-build-info.outputs.org-source }}
           started-at: ${{ steps.set_start_time.outputs.started-at }}
           ended-at: ${{ steps.set_end_time.outputs.ended-at }}
+
+      - name: Delete test graph
```

Comment:
Wouldn't this create issues if there's multiple concurrent GHA test workflows running, and one workflow deletes a graph while the others are still running? This repo isn't updated often so maybe it's not worth working around it.

Reply:
Yep, that's right. Will remove. Didn't want to keep the graph around but maybe fine